java/src/plume/Intern.java

```diff
@@ -856,7 +857,7 @@ public SequenceAndIndices(T seq, int start, int end) {
     }
 
     @SuppressWarnings("unchecked")
-    /*@Pure*/ public boolean equals(/*@Nullable*/ Object other) {
+    /*@Pure*/ public boolean equals (/*>>>@GuardSatisfied SequenceAndIndices<T> this,*/ /*@GuardSatisfied*/ /*@Nullable*/ Object other) {
```

Comment:
Throughout, please don't insert space between method name and following parentheses.  That's incompatible with the current code style.  Please undo all such changes.